### PR TITLE
[EEITT-1256] - Fixing optionHelpText alignment and display bugs

### PR DESCRIPTION
**Pull request by paul-request**

### CODE REVIEW

File: app/uk/gov/hmrc/gform/views/form/snippets/choice.scala.html [hunk 1]
```diff
@@ -22,47 +22,66 @@
...
-                       type="@inputType"
-                       @if(optionHelpText(index).body.nonEmpty) {
-                       aria-controls="helptext-@{fieldValue.id}@index"
-                       aria-expanded="false"
-                       }
-                       @validationResult.flatMap(_.getOptionalCurrentValue(fieldValue.id.value +
-                       index.toString)).orElse(prepop.find(_== index.toString)).map(_=> "checked").getOrElse("") />
-                @option
-            </label>
+              @if(optionHelpText.nonEmpty && optionHelpText(index).body.nonEmpty) {
```
Comment (pawelpanasewicz) [inline app/uk/gov/hmrc/gform/views/form/snippets/choice.scala.html]: We should have it done in backend.

**Comment by pawelpanasewicz** [comment]
@paul-request , we will fix it in order to safe you from putting login into views.

File: app/uk/gov/hmrc/gform/views/form/snippets/choice.scala.html [hunk 2]
```diff
@@ -22,47 +22,66 @@
...
+                      @validationResult.flatMap(_.getOptionalCurrentValue(fieldValue.id.value +
+                      index.toString)).orElse(prepop.find(_== index.toString)).map(_=> "checked").getOrElse("") />
+                    @option
+                </label>
+              }
             }
+            </div>
+
+            @if(orientation.toString.toLowerCase.equals("horizontal")) {
+              @options.toList.zipWithIndex.map{ case (option, index) =>
```
[the remark this answers is not in the document]
Comment (paul-request) [reply app/uk/gov/hmrc/gform/views/form/snippets/choice.scala.html]: @DanielConnelly If `orientation === horizontal` then we have to render the optionHelpText under the labels. If `orientation === vertical` then we must render it under each label. (See line 37 and Pawel's comment)

**Comment by paul-request** [comment]
@pawelpanasewicz @DanielConnelly Is anyone working on this? Is there a ticket to fix this in the backend? Worried this has just slipped through the net and it does need to be fixed for MVP...

**Comment by pawelpanasewicz** [comment]
@DanielConnelly has it on his worktop

**Comment by pawelpanasewicz** [comment]
well, actually it wass done by @DanielConnelly in other commit. I'm closing this PR

**Pull request closed without merging**